src/components/Map.vue

```diff
@@ -59,6 +59,10 @@ onMounted(() => {
   }).addTo(map)
   updateAtlas()
   map.invalidateSize()
+
+  // This ensures that when the map is shown, it will clear the previous report data
+  // including community name, lat, lon, and data.
+  atlasStore.clearReport()
```

Comment:
Could we actually move this to an `unmount()` hook in the `Report` component?  It feels more self-contained there.